.github/docker/Dockerfile.centreon-collect-mariadb-alma9-test

```diff
@@ -29,7 +36,9 @@ dnf install --best -y \
     psmisc \
     perl-HTTP-Daemon-SSL \
     perl-JSON \
+    python3-setuptools \
```

Comment:
Is it used ? because it seems not installed in other docker images
Asking because new docker image is 40MB bigger: 
![image](https://github.com/user-attachments/assets/073b0d5d-7a8f-4c78-b247-1a08e370f2bd)

Reply:
Yes I wanted to use the same mariadb version in our tests than in centreon for alma9. And the packages provided by Mariadb need this package.